### PR TITLE
ClassificationReport missing plot

### LINKED ISSUE #102: Missing plot for ClassificationReport in the current docs
There is an example [here](https://sklearn-evaluation.readthedocs.io/en/latest/api/plot.html#sklearn_evaluation.plot.ClassificationReport) of the `ClassificationReport` but the actual plot is missing.

@neelasha23 @idomic 
We should check if it persists after migrating to new docs.

### PR DESCRIPTION
**Pull request by neelasha23**
Closes #102

### CODE REVIEW

File: examples/ClassificationReport.py [hunk 1]
```diff
@@ -4,27 +4,36 @@
...
 
 from sklearn_evaluation import plot
+import matplotlib.pyplot as plt
 
-X, y = datasets.make_classification(200, 10, n_informative=5, class_sep=0.65)
-X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3)
 
-y_pred_rf = RandomForestClassifier().fit(X_train, y_train).predict(X_test)
-y_pred_lr = LogisticRegression().fit(X_train, y_train).predict(X_test)
+def show_figure(fig):
```
Comment (edublancas) [inline examples/ClassificationReport.py:10]: any idea why this is necessary for this example but not for others?

Comment (neelasha23) [reply examples/ClassificationReport.py:10]: This method doesn't return `Axes` object like the other methods. 
https://github.com/ploomber/sklearn-evaluation/blob/8414baaec836b0d17cf7614061f8dcfd93415d13/src/sklearn_evaluation/plot/classification_report.py#L132

The class object returned has an attribute `figure`, but the statement: `cr_rf.figure.show()` throws error:
```
AttributeError: Figure.show works only for figures managed by pyplot, normally created by pyplot.figure()
```

Found a solution in this [thread](https://stackoverflow.com/questions/49503869/attributeerror-while-trying-to-load-the-pickled-matplotlib-figure)

Comment (neelasha23) [reply examples/ClassificationReport.py:10]: It seems this issue is with `ConfusionMatrix` and `ClassificationReport` but not `ROC`. Trying to figure out the difference

Comment (neelasha23) [reply examples/ClassificationReport.py]: The issue was with `Figure()`. replacing it with `matplotlib.pyplot.figure()` works

**Review by edublancas — APPROVED** [review]

File: examples/ClassificationReport.py (hunk 1 above)
Comment (idomic) [reply examples/ClassificationReport.py]: Nice catch!

**Pull request merged**